src/corytortoise/DuelsPE/Main.php

```diff
@@ -78,6 +81,19 @@ public function getPrefix() {
       $prefix = str_replace("&", "§", $prefix);
       return $prefix . " ";
     }
+	    
+    public function addNewCommands(){
+		$this->registerc(['DuelCommand'],new duel($this)); 
```

Comment:
oh sorry I was working on it then i got side tracked and starting playing minecraft *laughing emoji*